### PR TITLE
fix(table): fixed hoverable, selectable table

### LINKED ISSUE #3852: Selectable table follow ups
* Selected, not expanded rows don't have blue accent

Screen Shot 2021-02-10 at 4 29 20 PM

* Top shadow appears bigger than the bottom shadow. May have something to do with the tr's bottom border?

Screen Shot 2021-02-10 at 4 30 33 PM

* Can we just update `tr.pf-m-hoverable` and `tbody.pf-m-hoverable` to have a white background instead of setting it on hover, focus, active, and selected? If so, add a note to make that the default unless there is a reason not to.

* Borders aren't appearing correctly in the grid view - the expanded/selected border color is wrong and plain selected rows don't have a border at all.

### PR DESCRIPTION
**Pull request by mattnolting**
closes #3852

### CODE REVIEW

File: src/patternfly/components/Table/examples/Table.md [hunk 1]
```diff
@@ -1409,13 +1409,98 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
...
+    {{> table-tr--basic table-tr--basic--index="7" table-tr--IsSelected="true" table-tr--basic--title="<b>Selected</b>"}}
+    {{> table-tr--basic table-tr--basic--index="8"}}
+    {{> table-tr--basic table-tr--basic--index="9"}}
+    {{> table-tr--basic table-tr--basic--index="10" table-tr--basic--IsExpanded="true" table-tr--IsSelected="true" table-tr--basic--title="<b>Selected</b>"}}
+    {{> table-tr--basic table-tr--basic--index="11"}}
+  {{/wrapper}}
+{{/table}}
+```
+
+### Expandable, hoverable and selectable
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/Table.md:1448]: oxford comma ftw!

```suggestion
### Expandable, hoverable, and selectable
```

**Review by mcoker — COMMENTED** [review]
This looks great. Only thing I see is when you have adjacent selected rows, if you hover or focus on a row, the shadow should still apply to that row. Right now I only see a single shadow around a group of selected rows, and hovering/focusing on any selected row in the group doesn't show a shadow.

**Comment by mattnolting** [comment]
@mcoker updated!

**Review by mcoker — APPROVED** [review]
L🔥TM!

**Pull request merged**